docs/spec/database.md

```diff
@@ -3,17 +3,18 @@ topic: Database
 date: 2025-05-30
 discussion: https://github.com/orgs/k8tre/discussions/9
 k8tre-statements:
-  spec: K8TRE Specification-conformant apps shall allow the use of the default DB. A K8TRE should integrate with an organisation's existing databases where appropriate.
-  refimpl: The K8TRE Reference Implementation includes a default Postgres DB, for the general use of apps. CloudNativePG used with ArgoCD to configure and manage this  on-cluster DB.
+  spec: Databases should be treated as attached resources and may be deployed on the cluster, or apps may connect to off-cluster databases.
 ---
 
-### Database
+{{ spec_content(page.meta) }}
 
-*Questions*:
-1. What should K8TRE Specification say about *in-cluster* DBs and what should it say about *off-cluster* DBs?
-2. What is K8TRE Reference Implementation doing regarding DBs?
-3. How prescriptive should K8TRE Specification be in dictating how DB's are deployed and managed on-cluster?
+**Questions**:
+
+1. **What should K8TRE Specification say about *in-cluster* DBs and what should it say about *off-cluster* DBs?**
+
+    Databases should be attached resources, explicitly referenced. TRE administrators may use an externally provided database service, such as AWS RDS, but where applications can use an on-cluster database, they should consider using the CNPG operator to deploy an instance of Postgres DB, rather than using a different Postgres helm chart and introducing an additional a dependency.
```

Comment:
```suggestion
    Databases should be attached resources, explicitly referenced. TRE administrators may use an externally provided database service, such as AWS RDS, but where applications can use an on-cluster database, they should consider using the CNPG operator to deploy an instance of Postgres DB, rather than using a different Postgres helm chart which introduces an additional dependency.
```

Reply:
We should define what `attached resources` means somewhere (can be in a different PR)